requirements.txt

```diff
@@ -1,26 +1,27 @@
-alembic==0.9.6
-astroid==1.6.0
-blinker==1.4
-click==6.7
-csh-ldap==1.2.4.dev41
-Flask==0.12.2
-Flask-Migrate==2.1.1
-Flask-SQLAlchemy==2.3.2
-isort==4.2.15
-itsdangerous==0.24
-Jinja2==2.9.6
-lazy-object-proxy==1.3.1
-Mako==1.0.7
-MarkupSafe==1.0
-mccabe==0.6.1
-psycopg2==2.7.3.2
-pyldap==2.4.37
-pylint==1.8.1
-python-dateutil==2.6.1
-python-editor==1.0.3
-raven==6.3.0
-six==1.11.0
-SQLAlchemy==1.1.15
-structlog==17.2.0
-Werkzeug==0.12.2
-wrapt==1.10.11
+alembic
```

Comment:
Please don't remove the explicit versioning from these. If you'd like to update them to a known working version that's alright. It's been awhile since I've done that but if I recall correctly you'll want to have a clean virtualenv, install the dependencies, and run `pip freeze`.

Reply:
They honestly shouldn't be that out of date, I updated them when I updated pylint a month-ish ago.

Reply:
But yah: pip freeze > requirements.txt